File: Service_eHospital_DongNai_A/Implement/Zzz_Excel_ListBN_HuyetApService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class Zzz_Excel_ListBN_HuyetApService : BaseService<Zzz_Excel_ListBN_HuyetAp, IZzz_Excel_ListBN_HuyetApRepository>
    , IZzz_Excel_ListBN_HuyetApService
    {
    private readonly IZzz_Excel_ListBN_HuyetApRepository _Zzz_Excel_ListBN_HuyetApRepository;
    public Zzz_Excel_ListBN_HuyetApService(IZzz_Excel_ListBN_HuyetApRepository Zzz_Excel_ListBN_HuyetApRepository) : base(Zzz_Excel_ListBN_HuyetApRepository)
    {
    _Zzz_Excel_ListBN_HuyetApRepository = Zzz_Excel_ListBN_HuyetApRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/Zzz_Excel_ListBN_HuyetAp.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Zzz_Excel_ListBN_HuyetAp } from './Zzz_Excel_ListBN_HuyetAp.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Zzz_Excel_ListBN_HuyetApService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'MaYTe', 'TenBN', 'NamSinh', 'GioiTinh'];

    List: Zzz_Excel_ListBN_HuyetAp[] | undefined;
    ListFilter: Zzz_Excel_ListBN_HuyetAp[] | undefined;
    FormData!: Zzz_Excel_ListBN_HuyetAp;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Zzz_Excel_ListBN_HuyetAp";
    }
}

